src/test/obj_strdup/obj_strdup.c

```diff
@@ -150,6 +152,39 @@ do_strdup_null_alloc(PMEMobjpool *pop)
 	UT_ASSERT(!TOID_IS_NULL(wcs1));
 }
 
+/*
+ * do_strdup_uint64_range -- duplicate string with
+ * type number equal to range of unsigned long long int
+ */
+static void
+do_strdup_uint64_range(PMEMobjpool *pop)
+{
+	TOID(char) str1 = do_alloc(pop, TEST_STR_1, TYPE_SIMPLE_ALLOC_1);
+	TOID(char) str2 = do_alloc(pop, TEST_STR_2, TYPE_SIMPLE_ALLOC_1);
+	TOID(char) str3 = do_alloc(pop, TEST_STR_1, TYPE_SIMPLE_ALLOC_1);
+	TOID(char) str4 = do_alloc(pop, TEST_STR_2, TYPE_SIMPLE_ALLOC_1);
+	pmemobj_strdup(pop, &str1.oid, D_RO(str2), UINT64_MAX);
+	pmemobj_strdup(pop, &str3.oid, D_RO(str4), UINT64_MAX - 1);
+	UT_ASSERTeq(strcmp(D_RO(str1), D_RO(str2)), 0);
+	UT_ASSERTeq(strcmp(D_RO(str3), D_RO(str4)), 0);
+}
+
+/*
+ * do_strdup_alloc_empty_string -- duplicate string to internal container
+ * associated with type number equal to range of unsigned long long int
+ * and unsigned long long int - 1
+ */
+static void
+do_strdup_alloc_empty_string(PMEMobjpool *pop)
```

Comment:
the two `do_alloc` calls are enough, because they already call strdup and check the string.